### ISSUE TITLE
Sound error when sending a message in thunderbird, alpha-19414,3ccd7b85

### ISSUE THREAD

**Issue by dreinn** [nvaccess/nvda#10580]
### Actual behavior:

NVDA makes a sound error when pressing ctrl-enter for sending message in thunderbird.

### Expected behavior:

Sending message should be without error sound.

#### Windows version: Windows7

See attached log for info
[logSoundErrorThunderbird.txt](https://github.com/nvaccess/nvda/files/3925850/logSoundErrorThunderbird.txt)

**Comment by dreinn** [comment]
just for your information, this seems quite variable, no problem for last 5 messages I sent!

**Comment by Brian1Gaff** [comment]
What do you see in the log, is it perhaps  a sluggishness which makes the 
info slow to be changed depending on the sending of an email?


I've seen similar things happen on web sites that are sluggish to be fully 
displayed in waterfox and firefox but nothing apparently comes of these 
errors.
 Brian
bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by Adriani90** [comment]
I cannot reproduce this in NVDA alpha 19420,a2c6f0f8 and Thunderbird 68.3.0.
@dreinn could you please try with the same versions like me and report back?

**Comment by Adriani90** [comment]
No update from the initial author and I cannot reproduce. meanwhile two updates have been published for Thunderbird and still not reproducible. Closing as works for me.
